.gitignore

```diff
@@ -4,6 +4,7 @@
 release/
 build/
 NoLimitCoin-qt
+Makefile
```

Comment:
Why remove the Makefile from tracking?  This one not used? Should it be removed instead?

Reply:
It is not required. Makefile is supposed to be made on the fly using qmake. For someone who is building this on multiple platforms/machine with different versions of libraries, it does not make sense to keep it here. It's just like the release, everyone who is building it from source, can build it.

Reply:
It is not required. Makefile is supposed to be made on the fly using qmake. For someone who is building this on multiple platforms/machine with different versions of libraries, it does not make sense to keep it here. It's just like the release, everyone who is building it from source, can build it.